tcms/testcases/views.py

```diff
@@ -630,32 +662,20 @@ def get(self, request, case_id):
         self.plan_id = int(plan_id) if plan_id is not None else None
 
         this_cls = TestCaseCaseRunListPaneView
-        return super(this_cls, self).get(request, case_id)
+        return super(this_cls, self).get(request, self.case_id)
```

Comment:
Use `super()`

Reply:
Does pylint complain if we use case_id as parameter instead of self.case_id ? If there's no difference I prefer a smaller commit and not including lines which don't fix any issues or not change behavior.